shared/src/iosTest/kotlin/com/twilio/passkeys/mocks/PayloadMocks.kt

```diff
@@ -0,0 +1,248 @@
+package com.twilio.passkeys.mocks
+
+import com.twilio.passkeys.models.AuthenticatePasskeyRequest
+import com.twilio.passkeys.models.AuthenticatePasskeyRequestPublicKey
+import com.twilio.passkeys.models.AuthenticatePasskeyResponse
+import com.twilio.passkeys.models.AuthenticatorSelection
+import com.twilio.passkeys.models.CreatePasskeyRequest
+import com.twilio.passkeys.models.CreatePasskeyResponse
+import com.twilio.passkeys.models.KeyCredential
+import com.twilio.passkeys.models.PubKeyCredParams
+import com.twilio.passkeys.models.Rp
+import com.twilio.passkeys.models.User
+
+val createPasskeyChallengePayload =
+  """
+  {
+    "rp": {
+      "id": "example.com",
+      "name": "Example"
+    },
+    "user": {
+      "id": "WUU0ZmQzYWFmNGU0NTMyNGQwZjNlMTM0NjA3YjIxOTEyYg",
```

Comment:
This can be flaged as a secret, could you please change them to a placeholder?

Reply:
Definitely 😅, added Placeholder prefix just to avoid any confusion. But apart from that the length must be exact. Thank you